dataland-frontend/src/components/resources/frameworkDataSearch/lksg/LksgPanel.vue

```diff
@@ -47,11 +48,23 @@ export default defineComponent({
     companyId: {
       type: String,
     },
+    singleDataMetaInfoToDisplay: {
+      type: Object as () => DataMetaInformation,
+    },
   },
   watch: {
     companyId() {
-      this.listOfDataDateToDisplayAsColumns = [];
-      void this.fetchAllAcceptedDatasets();
+      console.log("companyId watcher executes in LksgPanel"); //TODO
```

Comment:
A lot of console logs in this file

Reply:
Will delete them now

Reply:
Deleted console logs, and also those in SfdrPanel